### DISCUSSION TITLE
Did this molecular dynamics job finish well?

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#703] [Q&A]
Hello,

I run a Molecular Dynamics job for a system with 4608 atoms. At the end of the output I got the following messages:

`Note: The following floating-point exceptions are signalling: IEEE_INVALID_FLAG IEEE_OVERFLOW_FLAG IEEE_UNDERFLOW_FLAG IEEE_DENORMAL`

and

```
ERROR] Runtime exception occurred
-5- xtb_calculator_singlepoint: Electronic structure method terminated
-4- scf: Setup of Coulomb evaluator failed
-3- type_latticepoint_update: Could not generate lattice points
-2- xtb_calculator_singlepoint: Electronic structure method terminated
-1- scf: Self consistent charge iterator did not converge
```

My doubt: Can I consider that this job finished well?

Regards,

Camps

**Comment by haneug** [comment]
The SCC simply did not converge and your MD job was aborted. I would not consider this a well-finished job. Do you see any hot spots in your last geometry? You could restart your MD on the final structure with a higher etemp or use the GFN-FF instead. Maybe smaller timesteps could also help?
